### PR TITLE
add OpenTracing Shim usage

### LINKED ISSUE #1396: Document usage of the OpenTracing Shim
The README in the module should have a how-to usage section in it.

### PR DESCRIPTION
**Pull request by malafeev**
fix #1396

### CODE REVIEW

File: opentracing_shim/README.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
+[![Javadocs][javadoc-image]][javadoc-url]
 # OpenTelemetry - OpenTracing Shim
+A bridge layer from OpenTelemetry to the OpenTracing API.
+It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.
+
+## Usage
+
+1. Configure OpenTelemetry tracer
```
Comment (carlosalberto) [inline opentracing_shim/README.md:8]: Can we mention we are talking about the global instance?

File: opentracing_shim/README.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+1. Create OpenTracing tracer from OpenTelemetry tracer
```
Comment (carlosalberto) [inline opentracing_shim/README.md:9]: Probably we should add an extra section on how to manually specify a `Tracer` with the rest of the components.

**Comment by carlosalberto** [comment]
Thanks for taking on this (I got reminded of adding an utility method to create AND register a shim :) )

File: opentracing_shim/README.md (hunk 1 above)
Comment (malafeev) [reply opentracing_shim/README.md]: done

File: opentracing_shim/README.md (hunk 2 above)
Comment (malafeev) [reply opentracing_shim/README.md]: done

File: opentracing_shim/README.md [hunk 3]
```diff
@@ -1,6 +1,24 @@
+[![Javadocs][javadoc-image]][javadoc-url]
 # OpenTelemetry - OpenTracing Shim
+A bridge layer from OpenTelemetry to the OpenTracing API.
+It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.
```
Comment (jkwatson) [inline opentracing_shim/README.md:4]:
```suggestion
The OpenTracing shim is a bridge layer from OpenTelemetry to the OpenTracing API.
It takes OpenTelemetry Tracer and exposes it as an implementation of an OpenTracing Tracer.
```

File: opentracing_shim/README.md [hunk 4]
```diff
@@ -1,6 +1,24 @@
...
 
-[![Javadocs][javadoc-image]][javadoc-url]
+## Usage
+
+There are 2 ways to expose OpenTracing tracer:
```
Comment (jkwatson) [inline opentracing_shim/README.md:8]:
```suggestion
There are 2 ways to expose an OpenTracing tracer: 
```

File: opentracing_shim/README.md [hunk 5]
```diff
@@ -1,6 +1,24 @@
+[![Javadocs][javadoc-image]][javadoc-url]
 # OpenTelemetry - OpenTracing Shim
+A bridge layer from OpenTelemetry to the OpenTracing API.
+It takes OpenTelemetry Tracer and expose it as an OpenTracing Tracer.
 
-[![Javadocs][javadoc-image]][javadoc-url]
+## Usage
+
+There are 2 ways to expose OpenTracing tracer: 
+1. From global OpenTelemetry configuration
```
Comment (jkwatson) [inline opentracing_shim/README.md:9]:
```suggestion
1. From the global OpenTelemetry configuration 
```

File: opentracing_shim/README.md [hunk 6]
```diff
@@ -1,6 +1,24 @@
...
 
-[![Javadocs][javadoc-image]][javadoc-url]
+## Usage
+
+There are 2 ways to expose OpenTracing tracer: 
+1. From global OpenTelemetry configuration 
+    ```java
+    Tracer tracer = TraceShim.createTracerShim();
+    ```
+1. From provided `TracerProvider` and `CorrelationContextManager`
```
Comment (jkwatson) [inline opentracing_shim/README.md:13]:
```suggestion
1. From the provided `TracerProvider` and `CorrelationContextManager`:
```

File: opentracing_shim/README.md [hunk 7]
```diff
@@ -1,6 +1,24 @@
...
+    ```java
+    Tracer tracer = TraceShim.createTracerShim(tracerProvider, contextManager);
+    ```
+
+Optionally register tracer as the OpenTracing GlobalTracer
```
Comment (jkwatson) [inline opentracing_shim/README.md:18]:
```suggestion
Optionally register the tracer as the OpenTracing GlobalTracer:
```

**Comment by malafeev** [comment]
@jkwatson @carlosalberto your comments addressed

**Review by jkwatson — APPROVED** [review]
Thanks! Documentation is always appreciated!

**Review by carlosalberto — APPROVED** [review]

**Pull request merged**